### PR TITLE
feat: measures page

### PR DESCRIPTION
**Pull request by macintoshpie**
## Changes
- add `/measures/` page for viewing all measures from enumerations.json

Screen Shot 2021-06-18 at 8 11 24 AM

### CODE REVIEW

File: bsyncviewer/static/css/overrides.css [hunk 1]
```diff
@@ -103,6 +103,7 @@ footer .main-footer a {
 
 footer .main-footer p {
     font-size: 14px;
+}
```
Comment (macintoshpie) [inline bsyncviewer/static/css/overrides.css:106]: syntax error in css file

**Review by aviveiros11 — APPROVED** [review]
This looks good 👍

File: bsyncviewer/static/css/overrides.css (hunk 1 above)
Comment (nllong) [reply bsyncviewer/static/css/overrides.css:106]: nice catch!

File: bsyncviewer/views.py [hunk 2]
```diff
@@ -357,7 +357,40 @@ def enumerations(request, version):
...
+def measures(request, version):
+    try:
+        schema = Schema.objects.get(version=version)
+    except BaseException:
+        raise Http404('Schema version provided does not exist.')
+
+    if not schema.enumerations_file:
+        return HttpResponseRedirect(reverse_lazy('measures', args=[DEFAULT_SCHEMA_VERSION]))
+
+    enumerations_data = json.load(schema.enumerations_file)
```
Comment (nllong) [inline bsyncviewer/views.py:376]: sweet! enumeration files look to now be in the database!

**Review by nllong — APPROVED** [review]
looks swell!

**Comment by nllong** [comment]
@macintoshpie --- did you add this to the header of the site?

**Comment by kflemin** [comment]
@nllong I think this will be linked from the /releases page

**Comment by nllong** [comment]
> @nllong I think this will be linked from the /releases page

sweet, that works!

**Pull request merged**